blocks/triple-chain-block/themes/news.json

```diff
@@ -29,6 +29,15 @@
 			},
 			"desktop": {}
 		}
+	},"triple-chain-child-item-grid": {
```

Comment:
There should be a new line between `},` and `"triple-chain-child-item-grid"`.

Reply:
@vgalatro Added new line